homework/сlass_hierarchy_of_animals.py

```diff
@@ -0,0 +1,57 @@
+# 1. Create a class hierarchy of animals with at
+#    least 5 animals that have additional methods each.
+
+class Carnivora:
+    sound: str
+
+    def __init__(self):
+        self.name = input('Name of animal: ')
```

Comment:
do not use input here, pass name as a parameter to __init__ method

Reply:
Why I shouldn't use it here?
fixed